### PR TITLE
fix lua json require statements

### PR DESCRIPTION
**Pull request by Nick-Mazuk**
Fixes the bug noticed in https://github.com/finale-lua/lua-scripts/issues/343.

Though let's leave that issue open for now since the long-term solution is to update the bundler to support that syntax for require.

Feel free to merge when ready.

### CODE REVIEW

**Review by rpatters1 — APPROVED** [review]
Did you check if any of those subfiles have offending require statements? Otherwise, lgtm.

**Comment by amelchi** [comment]
sorry I am not a developer...
what should I do? :)
thanks

**Comment by rpatters1** [comment]
As soon as this Pull Request is merged, you should be able to get the new version and it should work.

**Comment by Nick-Mazuk** [comment]
Yes, none of the subfiles have any require statements. Merging.

**Pull request merged**
